workspaces/playlist/plugins/playlist-backend/src/service/router.ts

```diff
@@ -270,6 +266,5 @@ export async function createRouter(
     res.status(200).end();
   });
 
-  router.use(errorHandler());
```

Comment:
ah I think this should actually be replaced by
```ts
import { MiddlewareFactory } from '@backstage/backend-defaults/rootHttpRouter';

router.use(MiddlewareFactory.create({ config, logger }).error());
```
to fix the failing tests

Reply:
Ok, giving that a try, needed a few changes for that. I removed it completely as the current backend plugin template code doesn't add it anymore, assumed it wasn't needed: https://github.com/backstage/backstage/blob/994506952c20d7708c2f8a5f6498b19132157166/packages/cli/templates/default-backend-plugin/src/router.ts#L50